Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -508,8 +508,8 @@ LoaderSamples FlacLoaderPlugin::next_frame()
             i64 side = current_subframes[1][i];
             mid *= 2;
             // prevent integer division errors
-            samples[i] = { static_cast<float>((mid + side) * .5f) * sample_rescale,
-                static_cast<float>((mid - side) * .5f) * sample_rescale };
+            samples[i] = { static_cast<float>(static_cast<float>(mid + side) * .5f) * sample_rescale,
+                static_cast<float>(static_cast<float>(mid - side) * .5f) * sample_rescale };
         }
```

Comment:
This seems unintuitive, you still cast after the addition, so does this actually change anything, considering that the `* .5f` essentially is a `static_cast` in disguise

Reply:
Yeah, I should remove the outer cast. This was pointed out by clang-tidy.